### DISCUSSION TITLE
We need a logo!

### DISCUSSION THREAD

**Discussion by NotFenixio** [SnarpleDev/snazzle-sk#3] [General]
Self explanatory. 

We can use the old one I designed for the original Snazzle, or we can redesign it:

![favicon](https://github.com/SnarpleDev/snazzle-sk/assets/103071021/840fadaf-655e-49ef-9118-f6b03d7f3afa)

**Comment by redstone-dev** [comment]
I've already made one. It's in the readme I believe

**Reply by NotFenixio** [reply]
How do we fit that into a 16x16 space

**Reply by redstone-dev** [reply]
good point, I shall use yours since I have nothing else to use :P
